src/Microsoft.OData.Core/Json/ODataJsonWriterUtils.cs

```diff
@@ -348,7 +369,7 @@ internal static void ODataValueToString(StringBuilder sb, ODataValue value)
         private static void ODataCollectionValueToString(StringBuilder sb, ODataCollectionValue value)
         {
             bool isFirst = true;
-            sb.Append("[");
+            sb.Append('[');
             foreach (object item in value.Items)

```

Comment:
why this change?

Reply:
Apparently, if you're adding a character to the string builder, passing it as a character gives you better performance than as a string.